modules/hybridBidAdapter.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+
+**Module Name**: Hybrid.ai Bidder Adapter
+**Module Type**: Bidder Adapter
+**Maintainer**: prebid@hybrid.ai
+
+# Description
+
+You can use this adapter to get a bid from Hybrid.ai
+
+About us: https://hybrid.ai
+
+## Sample Banner Ad Unit
+
+```js
+var adUnits = [{
+    code: 'banner_ad_unit',
+    mediaTypes: {
+        banner: {
+            sizes: [[728, 90]]
+        }
+    },
+    bids: [{
+        bidder: "hybrid",
+        params: {
+            placement: "banner",                  // required
+            placeId: "5af45ad34d506ee7acad0c26"   // required
+        }
+    }]
+}];
+```
+
+## Sample Video Ad Unit
+
+```js
+var adUnits = [{
+    code: 'video_ad_unit',
+    mediaTypes: {
+        video: {
+            context: 'outstream',    // required
+            playerSize: [640, 480]   // required
+        }
+    },
+    bids: [{
+        bidder: 'hybrid',
+        params: {
+            placement: "video",                   // required
```

Comment:
How is the adapter expected to handle multi-format ad Units?

Where is is both banner and video?

What would the placement be?

Was thinking placement could be just inferred based on the adUnit otherwise.

Reply:
Could you elaborate on this?

Reply:
The adapter doesn't support multi-format ad Units in the current realization. If this a required condition we can add support multi-format ad Units.